services/scan/src/workers/json_serialization.ts

```diff
@@ -205,10 +205,11 @@ export function deserialize(
 
         default:
           throwIllegalValue(data, '__dataType__');
+          break;
       }
+      break;
     }
 
-    // eslint-disable-next-line no-fallthrough
```

Comment:
This change was kinda unrelated. @jonahkagan changed it recently so TS will not delete unreachable code, so the `break;` statements can be added above to make ESLint happy. TS knows they're unreachable, but ESLint doesn't.